docs/cloud-tutorials/cloud-cli.md

```diff
@@ -403,11 +403,49 @@ Rename your organization. Only the organization admin (the original creator of t
 Applications belonging to organizations are hosted at the URL `https://<organization-name>-<app-name>.cloud.dbos.dev/`, so renaming your organization changes your application URLs. The old URLs are no longer accessible.
 :::
 
+---
+
 ### `dbos-cloud org remove`
 
 **Description:**
 Remove a user from an organization. Only the organization admin (the original creator of the organization) can run this command.
 
 **Parameters:**
 - `<username>`: The user to remove from your organization.
----
\ No newline at end of file
+---
+
+## Workflow Management Commands
+
+### `dbos-cloud workflow list`
+
+**Description:**
+List workflows run by your application in JSON format ordered by recency (most recently started workflows last).
```

Comment:
Both cases "ordered by recency (most recent x last)" reads kludgy. This parenthetical definition only makes sense if you want to introduce a definition for "recency" and then use it many times below. Otherwise, it's less words to just say "ordered by most recent last."